js/app/dominantColor.js

```diff
@@ -1,31 +1,19 @@
-/* exported get_dominant_color, get_dominant_color_finish */
+/* exported get_dominant_color */
 
 const GdkPixbuf = imports.gi.GdkPixbuf;
 const Gio = imports.gi.Gio;
 
-const AsyncTask = imports.app.asyncTask;
 const EosKnowledgePrivate = imports.gi.EosKnowledgePrivate;
 
 // Note: this is a temporary solution until the color extraction is done
 // at build-time. The use of this function should be limited to specific cases,
 // to avoid performance penalties.
-function get_dominant_color (model, cancellable, callback) {
```

Comment:
I think the rationale for wrapping it in the AsyncTask was to make it easier to refactor to async in the future.